src/components/BrowserPreview/utils/getMissingEventStreams.ts

```diff
@@ -0,0 +1,20 @@
+import { SavedQueryListItem } from '../../../modules/queries';
+
+export const getMissingEventStreams = (
+  query: SavedQueryListItem,
```

Comment:
Please, consider renaming the argument to `savedQuery` - to avoid duplicate property access like `query.query`